subprojects/packagefiles/win-iconv/meson.build

```diff
@@ -27,7 +27,7 @@ elif get_option('default_library') == 'shared'
   c_lib_args += '-DMAKE_DLL'
 endif
 
-src_files = files('win_iconv.c')
+src_files = files('iconv.h', 'localcharset.h', 'win_iconv.c')
```

Comment:
Headers [don't need to be included](https://mesonbuild.com/FAQ.html#do-i-need-to-add-my-headers-to-the-sources-list-like-in-autotools) in library/executable sources.